app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -2636,7 +2636,10 @@ private function _setStockUseConfigFieldsValues($rowData)
     {
         $useConfigFields = [];
         foreach ($rowData as $key => $value) {
-            $useConfigName = self::INVENTORY_USE_CONFIG_PREFIX . $key;
+            $useConfigName = $key === 'enable_qty_increments'
+                ? 'use_config_enable_qty_inc'
```

Comment:
I would suggest using the static definition instead of the hardcoded 'enable_qty_increments' value. 

```
Magento\CatalogInventory\Api\Data\StockItemInterface::ENABLE_QTY_INCREMENTS
```